tests/regression/77-mem-oob/06-memset-memcpy-oob.c

```diff
@@ -0,0 +1,21 @@
+// PARAM: --set ana.activated[+] memOutOfBounds --enable ana.int.interval --disable warn.info
+// TODO: The "--disable warn.info" part is a temporary fix and needs to be removed once the MacOS CI job is fixed
+#include <stdlib.h>
+#include <string.h>
+
+int main(int argc, char const *argv[]) {
```

Comment:
It maybe be a good idea to also play around with sizes that are not constant here, that might have exposed some of the issues above.

Reply:
Absolutely agree. I added some further cases in https://github.com/goblint/analyzer/pull/1197/commits/87dadd265c5a8af8f8087307ef31aa41de51ff56